application/language/spanish/station_lang.php

```diff
@@ -8,35 +8,35 @@
-$lang['station_logbooks_create'] = "Create Station Logbook";
-$lang['station_logbooks_status'] = "Status";
-$lang['station_logbooks_link'] = "Link";
-$lang['station_logbooks_public_search'] = "Public Search";
-$lang['station_logbooks_set_active'] = "Set as Active Logbook";
-$lang['station_logbooks_active_logbook'] = "Active Logbook";
-$lang['station_logbooks_edit_logbook'] = "Edit Station Logbook";    // Full sentence will be generated 'Edit Station Logbook: [Logbook Name]'
-$lang['station_logbooks_confirm_delete'] = "Are you sure you want to delete the following station logbook? You must re-link any locations linked here to another logbook.: ";
-$lang['station_logbooks_view_public'] = "View Public Page for Logbook: ";
-$lang['station_logbooks_create_name'] = "Station Logbook Name";
-$lang['station_logbooks_create_name_hint'] = "You can call a station logbook anything.";
-$lang['station_logbooks_edit_name_hint'] = "Shortname for the station logbook. For example: Home Log (IO87IP)";
-$lang['station_logbooks_edit_name_update'] = "Update Station Logbook Name";
-$lang['station_logbooks_public_slug'] = "Public Slug";
-$lang['station_logbooks_public_slug_hint'] = "Setting a public slug allows you to share your logbook with anyone via a custom website address, this slug can contain letters & numbers only.";
-$lang['station_logbooks_public_slug_format1'] = "Later it looks like this:";
-$lang['station_logbooks_public_slug_format2'] = "[your slug]";
-$lang['station_logbooks_public_slug_input'] = "Type in Public Slug choice";
-$lang['station_logbooks_public_slug_visit'] = "Visit Public Page";
-$lang['station_logbooks_public_search_hint'] = "Enabling public search function offers a search input box on the public logbook page accessed via public slug. Search only covers this logbook.";
-$lang['station_logbooks_public_search_enabled'] = "Public search enabled";
-$lang['station_logbooks_select_avail_loc'] = "Select Available Station Locations";
-$lang['station_logbooks_link_loc'] = "Link Location";
-$lang['station_logbooks_linked_loc'] = "Linked Locations";
-$lang['station_logbooks_no_linked_loc'] = "No Linked Locations";
-$lang['station_logbooks_unlink_station_location'] = "Unlink Station Location";
+$lang['station_logbooks'] = "Libros de Guardia de Estación";
+$lang['station_logbooks_description_header'] = "¿Qué son Libros de Guardia de Estación";
+$lang['station_logbooks_description_text'] = "Los Libros de Guardia de Estación le permiten agrupar Localizaciones de Estación, observando todas las localizaciones en una sola sesión, desde las áreas del libro de guardia hasta las analíticas. Muy útil cuando está operando en múltiples localizaciones, pero está siendo parte del mismo Círculo DXCC o VUCC.";
```

Comment:
I would say "Muy útil cuando opera desde múltiples localizaciones, pero que són  parte del mismo círculo DXCC o VUCC". (Although "circle" seems a bit weird, even in the English original. I guess its short for DXCC entity or VUCC grid square).